package.nls.json

```diff
@@ -46,7 +44,7 @@
     "AWS.explorerNode.lambda.error": "Error loading Lambda resources",
     "AWS.explorerNode.signIn": "Connect to AWS...",
     "AWS.explorerNode.signIn.tooltip": "Connect to AWS using a credential profile",
-    "AWS.lambda.explorerTitle": "Lambda",
+    "AWS.lambda.explorerTitle": "Explorer",
```

Comment:
Should this be the full name `"AWS Explorer"`?

Reply:
As it is now, it renders as `AWS: EXPLORER`, changing it would render as `AWS: AWS EXPLORER`. 
I think we may as well start with the former.